tools/events-automation/README.md

```diff
@@ -1 +1,71 @@
-TODO: write this
\ No newline at end of file
+### Event Class:
```

Comment:
Please provide an intro to this solution -- e.g. some of what is in the PR description but more evergreen -- "the maintainer of the Events section is faced with manual work; this process automates generation of a draft events list using foo, bar and baz"